corehq/ex-submodules/casexml/apps/phone/xml.py

```diff
@@ -121,6 +121,17 @@ def get_registration_element(restore_user):
     return root
 
 
+def get_registration_element_for_case(restore_user, case_id):
```

Comment:
"for_case" seems a little ambiguous, maybe "for_touchforms" or whatever it is that needs the case_id?

Reply:
`for_touchforms` (or `for_sms`) wouldn't be exactly right because usually SMS will be using the normal restore route. We use this route in a special mode that allows SMS users to fill out a form as a case rather than as a mobile user, hence `for_case`. I agree this is a bit ambiguous but I can't really think of a better name since restoring *as* a case and restoring to get *cases* is kind of unavoidably conflated for naming

Reply:
Going to merge to get this in, will reconsider naming on the next pass

Reply:
sounds reasonable